container/Dockerfile.trtllm

```diff
@@ -628,6 +667,12 @@ COPY --chown=dynamo: --from=wheel_builder $NIXL_PREFIX $NIXL_PREFIX
 COPY --chown=dynamo: --from=wheel_builder /opt/nvidia/nvda_nixl/lib64/. ${NIXL_LIB_DIR}/
 COPY --chown=dynamo: --from=wheel_builder /opt/dynamo/dist/nixl/ /opt/dynamo/wheelhouse/nixl/
 COPY --chown=dynamo: --from=wheel_builder /workspace/nixl/build/src/bindings/python/nixl-meta/nixl-*.whl /opt/dynamo/wheelhouse/nixl/
+# Copy ffmpeg (or dummy)
+COPY --chown=dynamo: --from=wheel_builder /usr/local/include/ /usr/local/include/
```

Comment:
Should we be copying the entire folder here?

What is the image size impact after adding these COPY commands?

Reply:
For the headers, we have 7 subdirs we care about, so I can split it into 7 COPY. Right now to make the Dockerfile cleaner I just have 1 COPY but that also includes some unrelated headers from `libltdl` and `protobuf`.

Sizes look like this:
Headers: ~1.5M
Libs: ~20MB (after removing duplicates)